inst/templates/project-README

```diff
@@ -27,7 +27,7 @@ What is special about using `README.Rmd` instead of just `README.md`? You can in
 summary(cars)
 ```
 
-You'll still need to render `README.Rmd` regularly, to keep `README.md` up-to-date. `devtools::build_readme()` is handy for this. You could also use GitHub Actions to re-render `README.Rmd` every time you push. An example workflow can be found here: <https://github.com/r-lib/actions/tree/v1/examples>.
+You'll still need to render `README.Rmd` regularly, to keep `README.md` up-to-date.
```

Comment:
I even deleted the reference to `devtools::build_readme()` here, because it's not relevant to a non-package project and, in fact, I don't even think it will work. In a non-package project, folks can use whatever their normal workflow is to render `README.Rmd`.